app/templates/users/details.hbs

```diff
@@ -19,6 +19,13 @@
     {{t "users.details.last_on_goodcity"}} {{time-ago timeValue=model.lastDisconnected smallPrefix=false}}
   </div>
 
+  {{#if (is-and canDisableUsers isDisabledUser)}}
```

Comment:
if we can create computed property with meaningful name instead of using `and` condtion here would be much clearer 🤔  ?
Thoughts ?